rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -2536,45 +2513,29 @@ bool opcode_00e3(const ScriptArguments& args, const ScriptPlayer player, const S
 
 	opcode 00e4
 	@arg player 
-	@arg arg2 
-	@arg arg3 
-	@arg arg4 
-	@arg arg5 
+	@arg coord0 
+	@arg coord1 
 	@arg arg6 
 */
-bool opcode_00e4(const ScriptArguments& args, const ScriptPlayer player, const ScriptFloat arg2, const ScriptFloat arg3, const ScriptFloat arg4, const ScriptFloat arg5, const ScriptInt arg6) {
-	RW_UNIMPLEMENTED_OPCODE(0x00e4);
-	RW_UNUSED(player);
-	RW_UNUSED(arg2);
-	RW_UNUSED(arg3);
-	RW_UNUSED(arg4);
-	RW_UNUSED(arg5);
-	RW_UNUSED(arg6);
-	RW_UNUSED(args);
-	return false;
+bool opcode_00e4(const ScriptArguments& args, const ScriptPlayer player, const ScriptVec2 coord0, const ScriptVec2 coord1, const ScriptInt arg6) {
+	auto ply = player->getCharacter();
```

Comment:
`ply` is a shitty abbreviation for a character.
Other instances in this file too.

Beyond the scope?